templates/player.html

```diff
@@ -39,35 +39,83 @@
         <h2 class="pull-right">Potential: <span data-bind="text: player.ratings()[player.ratings().length - 1].pot"></span></h2>
       </div>
     </div>
-    <div class="row">
-      <div class="col-xs-4">
-        <strong>Physical</strong><br>
-        Height: <span data-bind="text: player.ratings()[player.ratings().length - 1].hgt"></span><br>
-        Strength: <span data-bind="text: player.ratings()[player.ratings().length - 1].stre"></span><br>
-        Speed: <span data-bind="text: player.ratings()[player.ratings().length - 1].spd"></span><br>
-        Jumping: <span data-bind="text: player.ratings()[player.ratings().length - 1].jmp"></span><br>
-        Endurance: <span data-bind="text: player.ratings()[player.ratings().length - 1].endu"></span>
-      </div>
-      <div class="col-xs-4">
-        <strong>Shooting</strong><br>
-        Inside: <span data-bind="text: player.ratings()[player.ratings().length - 1].ins"></span><br>
-        Dunks/Layups: <span data-bind="text: player.ratings()[player.ratings().length - 1].dnk"></span><br>
-        Free Throws: <span data-bind="text: player.ratings()[player.ratings().length - 1].ft"></span><br>
-        Two Pointers: <span data-bind="text: player.ratings()[player.ratings().length - 1].fg"></span><br>
-        Three Pointers: <span data-bind="text: player.ratings()[player.ratings().length - 1].tp"></span>
-      </div>
-      <div class="col-xs-4">
-        <strong>Skill</strong><br>
-        Blocks: <span data-bind="text: player.ratings()[player.ratings().length - 1].blk"></span><br>
-        Steals: <span data-bind="text: player.ratings()[player.ratings().length - 1].stl"></span><br>
-        Dribbling: <span data-bind="text: player.ratings()[player.ratings().length - 1].drb"></span><br>
-        Passing: <span data-bind="text: player.ratings()[player.ratings().length - 1].pss"></span><br>
-        Rebounding: <span data-bind="text: player.ratings()[player.ratings().length - 1].reb"></span>
+    <div role="tabpanel">
+      <div class="col-lg-offset-4">
+        <ul class="nav nav-pills" role="tablist">
+          <li role="presentation" class="active small"><a href="#basic" data-toggle="pill">Basic</a></li>
+          <li role="presentation" class="small"><a id="compositePill" href="#composite" data-toggle="pill">Composite</a></li>
+        </ul>
+      </div><p>&nbsp;</p>
```

Comment:
I think this would look better below the ratings and styled much more subtlely, basically so it looks more like a normal text link. If you would rather me just work on the UI, let me know and I'll take a shot at it (hopefully after you fix the other things I commented on!)